Apollo.xcodeproj/project.pbxproj

```diff
@@ -1010,27 +1010,31 @@
 			files = (
 			);
 			inputPaths = (
+				"$TEMP_DIR/apollo-lastrun",
```

Comment:
Out of curiosity, what does this do?

Reply:
sorry, I misunderstood about input file path. so this is not needed.